include/votca/xtp/rpa_eigensolution.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *            Copyright 2009-2018 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef _VOTCA_XTP_RPA_EIGENSOLUTION_H
+#define _VOTCA_XTP_RPA_EIGENSOLUTION_H
+
+namespace votca
+{
+namespace xtp
+{
```

Comment:
if it is just a struct you can make it a public struct in the rpa header. No need for a new file unless you want to extend it